tests/projects/local-kit/packages/pkg-d/Cargo.toml

```diff
@@ -1,3 +1,6 @@
+[workspace]
+members = []
```

Comment:
Why is this workspace entry needed?

Reply:
> Why is this workspace entry needed?

Because Twoliter has a workspace above it causing an error along the lines of "the package thinks its in a workspace when it isn't" when running a command (e.g. cargo metadata) in the pkg-d directory.

The fix is probably to require a global workspace for the whole of the Twoliter project, but that might interact poorly with a `sources` workspace(TBD). (And you would definitely want a separate `sources` workspace so that you can build your first-party Rust code, i.e. for development work, without all of the buildsys machinery in build.rs files taking place.